llama_index/program/predefined/evaporate.py

```diff
@@ -0,0 +1,111 @@
+import logging
```

Comment:
let's add this to `llama_index/program/__init__.py` (and import from `llama_index.program` in the notebook)